.github/workflows/continuous_integration.yml

```
@@ -91,7 +91,9 @@ jobs:
           sudo pip install qgis-plugin-ci
       - name: Release
         run: |
-          qgis-plugin-ci release ${GITHUB_REF##*/} --transifex-token ${TX_TOKEN} --github-token ${GITHUB_TOKEN} --osgeo-username ${OSGEO_USERNAME} --osgeo-password ${OSGEO_PASSWORD}
+          RELEASE_VERSION=${GITHUB_REF/refs\/tags\/v/}
+          RELEASE_TAG=${GITHUB_REF##*/}
+          qgis-plugin-ci release ${RELEASE_VERSION} --release-tag ${RELEASE_TAG} --transifex-token ${TX_TOKEN} --github-token ${GITHUB_TOKEN} --osgeo-username ${OSGEO_USERNAME} --osgeo-password ${OSGEO_PASSWORD}
```

```suggestion
          qgis-plugin-ci release ${RELEASE_VERSION} \
              --release-tag ${RELEASE_TAG} \
              --transifex-token ${TX_TOKEN} \
              --github-token ${GITHUB_TOKEN} \
              --osgeo-username ${OSGEO_USERNAME} \
              --osgeo-password ${OSGEO_PASSWORD}
```

Improve the readability even more.